suzieq/poller/worker/coalescer_launcher.py

```diff
@@ -64,7 +60,7 @@ async def start_and_monitor_coalescer(self):
         """
         try:
             await self._monitor_coalescer()
-        except CancelledError:
+        except asyncio.CancelledError:
```

Comment:
I had done the code the way it was deliberately. When we upgrade to 3.8 or whatever version supports exceptions the way you had originally done it, this would automatically work. Unless you object strongly, I suggest we go back to how it was (module import inside a try/except).